package.json

```diff
@@ -36,6 +36,11 @@
     "test:testcafe-setup": "mkdir -p test-reports/testcafe && yarn start:mockDuo --no-open",
     "test:testcafe-run": "wait-on http-get://localhost:3000 && testcafe -c 2 chrome:headless test/testcafe/spec/ --reporter xunit:build2/reports/junit/testcafe-results.xml",
     "test:testcafe-ci": "run-p -r test:testcafe-setup \"test:testcafe-run\" -- 2>/dev/null",
+    "test:testcafe-en-leaks-setup": "mkdir -p test-reports/testcafe && grunt exec:pseudo-loc && yarn build:dev && yarn start:dev-server --no-open",
+    "test:testcafe-run-en-leaks": "wait-on http-get://localhost:3000 && testcafe -c 2 chrome test/testcafe/enLeaks/",
+    "test:testcafe-run-en-leaks-ci": "wait-on http-get://localhost:3000 && testcafe -c 2 chrome:headless test/testcafe/enLeaks/ --reporter xunit:build2/reports/junit/testcafe-en-leaks-results.xml",
+    "test:enleaks-ci": "run-p -r test:testcafe-en-leaks-setup \"test:testcafe-run-en-leaks-ci\" -- 2>/dev/null",
+    "test:enleaks": "run-p -r test:testcafe-en-leaks-setup \"test:testcafe-run-en-leaks\" -- 2>/dev/null",
```

Comment:
We can probably clean this up a bit by chaining:
  - The `test:testcafe-setup` command calls `build:dev` and `start:dev-server`, so we should be able to run pseudo-loc before this executes

```suggestion
    "test:testcafe-run-en-leaks": "wait-on http-get://localhost:3000 && testcafe -c 2 chrome test/testcafe/enLeaks/",
    "test:testcafe-run-en-leaks-ci": "wait-on http-get://localhost:3000 && testcafe -c 2 chrome:headless test/testcafe/enLeaks/ --reporter xunit:build2/reports/junit/testcafe-en-leaks-results.xml",
    "test:enleaks-ci": "grunt exec:pseudo-loc && run-p -r test:testcafe-setup\"test:testcafe-run-en-leaks-ci\" -- 2>/dev/null",
    "test:enleaks": "grunt exec:pseudo-loc && run-p -r test:testcafe-setup \"test:testcafe-run-en-leaks\" -- 2>/dev/null",
```

Also, do you mind creating a follow-up ticket to move all this to `buildtools`? We should be able to pass arguments through instead of hardcoding paths and reporters. 🧹

Reply:
https://oktainc.atlassian.net/browse/OKTA-385965